MdeModulePkg/Core/Dxe/Event/Event.c

```diff
@@ -628,6 +628,7 @@ CoreCheckEvent (
   return Status;
 }
 
+// MU_CHANGE [BEGIN] - Support all TPLs

```

Comment:
In this scenario, the `git diff` is failing to accurately show the changes I made. What is actually happening in this PR is that I'm moving the core logic out of `WaitForEventInternal` and into `CoreWaitForEvent`, so I'm making large changes to both functions, but because it's more of a "move", the diff shows really weird. If you look at it as a whole file without the diff, you see the //MU_CHANGE wraps CoreWaitForEvent and CoreWaitForEventInternal.